lib/dag/edges.rb

```diff
@@ -1,59 +1,69 @@
 module Dag
   module Edges
-
     def self.included(base)
-      base.send :include, EdgeInstanceMethods
+      base.send(:include, EdgeInstanceMethods)
+    end
+
+    def ancestor_scoped_record_id(ancestor)
```

Comment:
Když se vytváří hrana stromu, doplňují se i nepřímé hrany. Během toho nesmím zapomenout předávat hodnotu `scoped_record_id`, aby hrany spadaly do jednoho stromu.
Podmínky níže by možná měly být opačně a možná by stačila jen 
`ancestor.public_send(scoped_record_id_column_name) if scoped_record_id_column_name`
To ještě prozkoumám.